tracer/src/Datadog.Trace/Agent/Transports/ApiWebRequestFactory.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using System.Collections.Generic;
 using System.Net;
+#pragma warning disable CS0618
```

Comment:
nit: it's nice to know what we're disabling 🙂 

```suggestion
#pragma warning disable CS0618 // WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead.